packages/ra-ui-materialui/src/list/DatagridHeaderCell.js

```diff
@@ -23,7 +23,7 @@ const useStyles = makeStyles(
     { name: 'RaDatagridHeaderCell' }
 );
 
-export const DatagridHeaderCell = ({
+const DatagridHeaderCell = ({
```

Comment:
I think this one has a HOC and should still be exported

Reply:
Done.